src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js

```diff
@@ -81,7 +81,6 @@ describe('DuplicateMappingProfileRoute', () => {
     });
 
     it('should have enabled save button if there are no changes', () => {
-      screen.debug(screen.getByRole('button', { name: 'Save & close' }));
```

Comment:
whoa, I wonder who left this here...

Reply:
But I eventually removed it. I tracked it down. By the way who removed the PR and allowed it to slip in?🙂

Reply:
Haha! I thought it was me, to be honest 😂

Reply:
😬